packages/design/src/shader-lab/Backend.ts

```diff
@@ -0,0 +1,4 @@
+export enum EBackend {
+  GLES100,
+  GLES300
+}
```

Comment:
- Why you define `EBackend` twice?
-  `EBackend` is not a good name

Reply:
done